static/app/components/forms/fieldGroup/index.tsx

```diff
@@ -58,28 +58,13 @@ function FieldGroup({
     return null;
   }
 
+  const controlStateElement = controlState ?? (
```

Comment:
Drive-by PSA that we should start inlining this. It's annoying to read the render functions of components by having to jump to these definitions (especially as components get larger)

Reply:
I agree with @JonasBa  it would be better if this were inline